pkg/bundle/files.go

```diff
@@ -4,42 +4,80 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"sync"
 
 	"github.com/RedHatGov/bundle/pkg/config/v1alpha1"
+	"github.com/gammazero/workerpool"
+	"github.com/sirupsen/logrus"
 )
 
+type Reconciler struct {
+	newFiles   map[string]v1alpha1.File
+	foundFiles map[string]struct{}
+	mu         *sync.Mutex
```

Comment:
Since `Reconciler`'s methods have a pointer receiver, I'd use a `sync.Mutex` value instead. Also this can be a `sync.RWMutex` with the below suggested changes.
```suggestion
	mu         sync.RWMutex
```

Reply:
Changed added. Resolving.